app/src/org/commcare/views/widgets/SpinnerMultiWidget.java

```diff
@@ -111,7 +111,7 @@ public void onClick(DialogInterface dialog, int which, boolean isChecked) {
                                 selections[which] = isChecked;
 
                                 if (hasListener()) {
-                                    widgetChangedListener.widgetEntryChanged();
+                                    widgetChangedListener.widgetEntryChanged(SpinnerMultiWidget.this);
                                 }
```

Comment:
I'm wondering: can every insteance of `if (hasListener()) {
widgetChangedListener.widgetEntryChanged(this); }` can just be replaced by a call to `QuestionWidget.widgetEntryChanged()`?